cmd/noderesourcetopology-plugin/main_test.go

```diff
@@ -152,7 +152,7 @@ profiles:
 			registryOptions: []app.Option{app.WithPlugin(noderesourcetopology.Name, noderesourcetopology.New)},
 			wantPlugins: map[string]*config.Plugins{
 				"default-scheduler": {
-					PreEnqueue: config.PluginSet{Enabled: []config.Plugin{{Name: schedulinggates.Name}}},
+					PreEnqueue: config.PluginSet{Enabled: []config.Plugin{{Name: schedulinggates.Name}, {Name: "DefaultPreemption"}}},
```

Comment:
how did scheduler-plugin u/s fix this problem? can we just resync again (= fetch more changes from u/s) instead of solving on our own?

Reply:
the resync happened againstthe most recent master branch and cmd/noderesourcetopology-plugin doesn't exist upstream so we need KNI special commit for this

Reply:
ok, fair enough